### PR TITLE
Define menu, tools, dashboard links in manifest files

### LINKED ISSUE #1859: Define components in manifest.json files
Instead of just tools, we need a way to define components in a flexible manner:

 * Main menu components
 * Tools
 * Dashboards

### PR DESCRIPTION
**Pull request by stefwalter**
Fixes #1859 

You can see how these are defined in #1859.

### CODE REVIEW

**Comment by stefwalter** [comment]
Needs documentation.

Depends on #1869

**Comment by stefwalter** [comment]
Documentation done.

**Comment by stefwalter** [comment]
Didn't document "dashboard" because I think those should only be implemented in Cockpit sparingly for now.

File: pkg/kubernetes/manifest.json [hunk 1]
```diff
@@ -2,9 +2,17 @@
     "version": 0,
     "discovery": [ "kubernetes" ],
 
+    "dashboard": {
+        "cluster": {
+            "path": "kubernetes/tool",
+            "label": "Dashboard",
+            "order": 0
+        }
```
Comment (mvollmer) [inline pkg/kubernetes/manifest.json]: This should probably be "Cluster" and 1.  It's not installed, but I had it linked into ~/.local and thus got a confusing navigation bar.

Comment (mvollmer) [inline pkg/kubernetes/manifest.json]: I'll fix this and merge.

Comment (stefwalter) [inline pkg/kubernetes/manifest.json]: Thanks for catching that.

**Pull request closed without merging**